### PR TITLE
Check if wazuh ports is open before to install the agent

### PR DESCRIPTION
**Pull request by mmsilveira**
## What?
Check if wazuh ports 1515 and 1514 is open before to installing the agent.

## Why?
Those ports are necessary to register and communicate to Wazuh server, so if any is inaccessible the agent doesn't register and reinstalling the agent will be necessary.

## How?
Create a ansible target to verify if the ports is open.

### CODE REVIEW

File: tasks/check.yml [hunk 1]
```diff
@@ -30,6 +30,39 @@
       meta: end_host
   when: wazuh_manager_ip is undefined or wazuh_manager_ip|length == 0
 
+- name: Check Wazuh manager port numbers are accessible from the current host
```
Comment (vitor-tyburski) [inline tasks/check.yml:33]:
```suggestion
- name: Check Wazuh manager port number is accessible from current host
```

File: tasks/check.yml [hunk 2]
```diff
@@ -30,6 +30,39 @@
...
+    host: "{{ wazuh_manager_ip }}"
+    port: "{{ wazuh_manager_port }}"
+    state: started         # Port should be open
+    delay: 0               # No wait before first check (sec)
+    timeout: 3             # Stop checking after timeout (sec)
+  ignore_errors: yes
+  check_mode: no           # wait_for not work with check_mode
+  register: check_wazuh_manager_ports
+
+- name: Check Wazuh registration port numbers are accessible from the current host
```
Comment (vitor-tyburski) [inline tasks/check.yml:44]:
```suggestion
- name: Check Wazuh registration port number is accessible from the current host
```

File: tasks/check.yml [hunk 3]
```diff
@@ -30,6 +30,39 @@
...
+    state: started         # Port should be open
+    delay: 0               # No wait before first check (sec)
+    timeout: 3             # Stop checking after timeout (sec)
+  ignore_errors: yes
+  check_mode: no           # wait_for not work with check_mode
+  register: check_wazuh_registration_ports
+
+- name: Test Wazuh manager and registration port
+  block:
+    - name: Test Wazuh manager or registration port are not accessible!
```
Comment (vitor-tyburski) [inline tasks/check.yml:57]:
```suggestion
    - name: Test Wazuh manager and registration port are accessible!
```

**Comment by marcelokkruger** [comment]
@mmsilveira, you have added a new check scenario. In this way you need to check the tests already created in the molecule, and also create a new test for this scenario. As this role is open, we have to guarantee the tests to maintain the quality.

**Comment by mmsilveira** [comment]
> @mmsilveira, you have added a new check scenario. In this way you need to check the tests already created in the molecule, and also create a new test for this scenario. As this role is open, we have to guarantee the tests to maintain the quality.

I have created a new test scenario to check wazuh open ports and changed the other tests to keep working. 

@vitor-tyburski and @marcelokkruger Can you check this MR again please?

**Comment by marcelokkruger** [comment]
Perfect man!

**Pull request merged**